config/metac.yaml

```diff
@@ -148,37 +148,22 @@ spec:
       resource: daemonsets
       updateStrategy:
         method: InPlace
-      labelSelector:
-        matchExpressions:
-          - {key: openebs-upgrade.dao.mayadata.io/managed, operator: DoesNotExist}
     - apiVersion: apps/v1
       resource: deployments
       updateStrategy:
         method: InPlace
-      labelSelector:
-        matchExpressions:
-          - {key: openebs-upgrade.dao.mayadata.io/managed, operator: DoesNotExist}
     - apiVersion: apps/v1
       resource: statefulsets
       updateStrategy:
         method: InPlace
-      labelSelector:
-        matchExpressions:
-          - {key: openebs-upgrade.dao.mayadata.io/managed, operator: DoesNotExist}
     - apiVersion: v1
       resource: configmaps
       updateStrategy:
         method: InPlace
-      labelSelector:
-        matchExpressions:
-          - {key: openebs-upgrade.dao.mayadata.io/managed, operator: DoesNotExist}
     - apiVersion: v1
       resource: services
       updateStrategy:
         method: InPlace
-      labelSelector:
-        matchExpressions:
-          - {key: openebs-upgrade.dao.mayadata.io/managed, operator: DoesNotExist}
```

Comment:
I think this label has not been yet pushed to production. So, this removal code is to address on staging?

Reply:
Due to having above labelSelector, the controller only lists the resources which do not have this label @avishnu, we are removing this in order to support the cases where the user may have installed OpenEBS using `openebs-upgrade` only and they may have removed the OpenEBS CR and if in that case, the user comes for adoption then the components will already have these labels.